pages/thank-you.jsx

```diff
@@ -37,9 +39,9 @@ var ThankYou = React.createClass({
           <div>
             <div id="header-copy">
               <div className="row">
-                <h1>From all of us at Mozilla</h1>
+                <h1><L10nMSG msgid="ty.From_all_of_us_at_Mozilla"/></h1>
```

Comment:
You mentioned there was an html formatted API call here, just a reminder to use it.